source/server/filter_chain_manager_impl.cc

```diff
@@ -558,42 +558,40 @@ const Network::FilterChain* FilterChainManagerImpl::findFilterChainForSourceIpAn
 }
 
 void FilterChainManagerImpl::convertIPsToTries() {
-  for (auto& port : destination_ports_map_) {
+  for (auto& [_, destination_ips_pair] : destination_ports_map_) {
```

Comment:
I believe the use of '_' as unused variable isn't really a C++ thing as in some other languages. With that being said, there isn't anything to stop you from naming the unused variable as '_'. But I would suggest naming the variable and not using it as '_' is not a wildcard in C++ and behaves differently than you might have hoped.

Reply:
Good to know this, will update.